Patches/Vanilla Weapons Expanded/Ammo/FireExtinguisher.xml

```diff
@@ -1,162 +1,162 @@
-                <graphicClass>Graphic_Single</graphicClass>
-                <shaderType>TransparentPostLight</shaderType>
-              </graphicData>
-              <projectile Class="CombatExtended.ProjectilePropertiesCE">
-                <damageDef>VWE_Extinguish</damageDef>
-                <damageAmountBase>50</damageAmountBase>
-                <explosionRadius>1</explosionRadius>
-                <preExplosionSpawnThingDef>Filth_FireFoam</preExplosionSpawnThingDef>
-                <preExplosionSpawnChance>1</preExplosionSpawnChance>
-                <soundExplode>VWE_Explosion_FireExtinguisher</soundExplode>
-                <applyDamageToExplosionCellsNeighbors>true</applyDamageToExplosionCellsNeighbors>
-                <speed>30</speed>
-                <flyOverhead>false</flyOverhead>
-              </projectile>
-            </ThingDef>
-          </value>
-        </li>
-
-        <!-- === Recipe === -->
-        <li Class="PatchOperationAdd">
-          <xpath>Defs</xpath>
-
-          <value>
-            <RecipeDef Name="ExtinguisherAmmoRecipeBase" ParentName="AmmoRecipeBase" Abstract="true">
-              <workSpeedStat>DrugSynthesisSpeed</workSpeedStat>
-              <effectWorking>Cook</effectWorking>
-              <soundWorking>Recipe_CookMeal</soundWorking>
-              <workSkill>Intellectual</workSkill>
-            </RecipeDef>
-          </value>
-        </li>
-
-        <li Class="PatchOperationAdd">
-          <xpath>Defs</xpath>
-
-          <value>
-            <RecipeDef ParentName="ExtinguisherAmmoRecipeBase">
-              <defName>MakeAmmo_VWE_Extinguisher</defName>
-              <label>make phosphate x400</label>
-              <description>Craft 400 units of Phosphate.</description>
-              <jobString>Making Phosphate.</jobString>
-              <ingredients>
-                <li>
-                  <filter>
-                    <thingDefs>
-                      <li>Chemfuel</li>
-                    </thingDefs>
-                  </filter>
-                  <count>20</count>
-                </li>
-                <li>
-                  <filter>
-                    <categories>
-                      <li>MeatRaw</li>
-                    </categories>
-                  </filter>
-                  <count>20</count>
-                </li>
-              </ingredients>
-              <fixedIngredientFilter>
-                <thingDefs>
-                  <li>Chemfuel</li>
-                </thingDefs>
-                <categories>
-                  <li>MeatRaw</li>
-                </categories>
-              </fixedIngredientFilter>
-              <products>
-                <Ammo_VWE_Extinguisher>500</Ammo_VWE_Extinguisher>
-              </products>
-            </RecipeDef>
-          </value>
-        </li>
-      </operations>
-    </match>
-  </Operation>
-
+<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
The diff is on the entire file. I suspect that you've inadvertedly changed the end-of-line marker. Could you commit the file with LF end-of-line markers instead of CRLF?

How to do this depends on your editor and how you've configured Git. See this answer on info on how to do so: https://stackoverflow.com/a/12876379/871050

Reply:
Replaced CRLF to LF using Notepad++, now it compares correctly.